internal/pkg/cli/pipeline_status.go

```diff
@@ -0,0 +1,87 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cli
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+type pipelineStatusVars struct {
+	*GlobalOpts
+	shouldOutputJSON bool
+	appName          string
+	pipelineName     string
+}
+
+type pipelineStatusOpts struct {
+	pipelineStatusVars
+
+	statusDescriber     pipelineStatusDescriber
+	initStatusDescriber func(opts *pipelineStatusOpts) error
+}
+
+func newPipelineStatusOpts(vars pipelineStatusVars) (*pipelineStatusOpts, error) {
+	return &pipelineStatusOpts{
+		pipelineStatusVars: vars,
+	}, nil
+}
+
+// Validate returns an error if the values provided by the user are invalid.
+func (o *pipelineStatusOpts) Validate() error {
+	return nil
+}
+
+// Ask prompts for fields that are required but not passed in.
+func (o *pipelineStatusOpts) Ask() error {
+	return nil
+}
+
+// Execute displays the status of the pipeline.
+func (o *pipelineStatusOpts) Execute() error {
+	err := o.initStatusDescriber(o)
+	if err != nil {
+		return fmt.Errorf("describe status of pipeline : ")
+	}
+	pipelineStatus, err := o.statusDescriber.Describe()
+	if err != nil {
+		return fmt.Errorf("describe status of pipeline : ")
+	}
+	fmt.Print(pipelineStatus)
+	return nil
+}
+
+// BuildPipelineStatusCmd builds the command for showing the status of a deployed pipeline.
+func BuildPipelineStatusCmd() *cobra.Command {
+	vars := pipelineStatusVars{
+		GlobalOpts: NewGlobalOpts(),
+	}
+	cmd := &cobra.Command{
+		Hidden: true, // TODO: remove when ready for production!
+		Use:    "status",
+		Short:  "Show the status of your application's pipeline.",
+		Long:   "Show the status of each deployed pipeline's stages.",
```

Comment:
```suggestion
		Long:   "Show the status of each stage in your pipeline.",
```

Reply:
Ahh, much better. Thanks! But "stage IN" or "stage OF"?

Reply:
Oh maybe of sounds better? Either's fine with me! Your choice.